setup.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import setuptools
+
+# Importing Optimus Prime Version
+import db-assessment.version
+
+name = "oracle-db-assessment"
+description = "A tool to enable collection of data from Oracle datases for homogeneous and heterogeneous database migration assessment"
+version = "0.1.0"
+release_status = "Development Status :: 4 - Beta"
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+dependencies = []
+with open("requirements.txt", "r") as fp:
+    for line in fp.readlines():
+        if not line.strip().startswith("#"):
+            dependencies.append(line.strip())
+
+extras_require = {}
+
+packages = setuptools.find_packages()
+
+packages += []
```

Comment:
Redundant statement

Reply:
That was included in the prior version you reviewed.